flink-runtime/src/test/java/org/apache/flink/runtime/state/StateBackendTestBase.java

```diff
@@ -711,14 +697,14 @@ public void testBackendUsesRegisteredKryoSerializer() throws Exception {
                 }
             }
 
-            assertTrue("Didn't see the expected Kryo exception.", numExceptions > 0);
+            assertThat(numExceptions > 0).as("Didn't see the expected Kryo exception.").isTrue();
         } finally {
             IOUtils.closeQuietly(backend);
             backend.dispose();
         }
     }
 
-    @Test
+    @TestTemplate
     @SuppressWarnings("unchecked")
     public void testBackendUsesRegisteredKryoSerializerUsingGetOrCreate() throws Exception {
```

Comment:
```suggestion
    void testBackendUsesRegisteredKryoSerializerUsingGetOrCreate() throws Exception {
```
could be package private